packages/block-editor/src/components/iframe/index.js

```diff
@@ -243,7 +239,6 @@ function Iframe( {
 	const bodyRef = useMergeRefs( [
 		useBubbleEvents( iframeDocument ),
 		contentRef,
-		clearerRef,
```

Comment:
This probably doesn't make a difference? Not sure why it's both added here and `ExperimentalBlockCanvas`. Maybe we should keep one, not sure why it was ever added to `ExperimentalBlockCanvas`.

Reply:
`ExperimentalBlockCanvas` adds the clearer as the `contentRef`, which is added to the same element through the line above this one. I don't get what the difference is? Is there a bug because it is added twice to the same element?

Reply:
I believe the problem is that the removed ref callback was attached to the `html`, which was causing the bug.

As far as I can tell, this doesn't introduce any regressions, so let's merge it.